apps/passport-client/components/screens/AddScreen/AddScreen.tsx

```diff
@@ -26,6 +30,7 @@ import { ProveAndAddScreen } from "./ProveAndAddScreen";
  */
 export function AddScreen() {
   useSyncE2EEStorage();
+  useRequirePassword();
```

Comment:
this may end up being fragile medium term. eg. we add another way for someone to get a PCD into their zupass and then forget to use this hook there.

Reply:
I also added it to the redux "add-pcd" action in https://github.com/proofcarryingdata/zupass/pull/738/commits/0ce49abc0e98bd5d6e9f599b4928c0bf7252d632